omps-integration.Dockerfile

```diff
@@ -0,0 +1,17 @@
+ARG OMPS_VERSION=latest
+FROM quay.io/operator-manifests/omps:${OMPS_VERSION}
```

Comment:
A prow CI job in the OMPS repo should run to build and push this image. Should this build always be from OMPS `master`?